Assets/Scripts/Movement.cs

```diff
@@ -2,9 +2,17 @@
 
 public class Movement : MonoBehaviour
 {
-    private float _velocity = 10f;
-    [SerializeField] private float _jumpH = 4f;
     Rigidbody2D _rb2D;
+    [SerializeField] private float _velocity = 10f;
+    [SerializeField] private float _JumpH = 10f;  
```

Comment:
Use camel case, this means this should be _jumpH, and also use something more describing, like _jumpHeight